docs/basics/SettingUpTheMiddleware.md

```diff
@@ -100,22 +99,3 @@ Within the body of the function based to the `catchError` operator, you can log
 Note that in the example above, the `console.error` function is not supported in IE 8/9.
 
 Also note that restarting the root epic can have some unintended consequences, especially if your application uses stateful epics, as they may lose state in the restart.
-
-## Redux DevTools
```

Comment:
I removed this whole sections since `configureStore` does that automatically and people with the knowledge to actively decide to keep using `createStore` for some reason should already be aware on how to set up the devtools - it is not redux-observable specific in the first place.